bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -323,7 +323,7 @@ private Addon convertTopicItemToAddon(DiscourseTopicItem topic, List<DiscourseUs
                 "posts_count", postsCount, //
                 "tags", tags.toArray(String[]::new));
 
-        // try to use an handler to determine if the add-on is installed
+        // try to use a handler to determine if the add-on is installed
```

Comment:
The rules for a/an in words that start with "h" are complicated and inconsistent. See https://editingandwritingservices.com/a-or-an-before-words-beginning-with-h/ for one discussion.

In this case, which to use probably depends on which English accent you are thinking of. The most American English accents would pronounce the "h" so "a" would be correct. But some accents would not sound the "h" (in speech it would come across as "andler") in which case "an" would be correct.

I would use "a" here, but I speak with an American accent (western US accent to be specific). I don't know if a UK English or many of the other native speakers would choose differently.